src/OrchardCore/OrchardCore.OpenId.Core/YesSql/Migrations/OpenIdMigrations.cs

```diff
@@ -62,21 +70,24 @@ public int UpdateFrom1()
 
         public int UpdateFrom2()
         {
-            SchemaBuilder.DropReduceIndexTable("OpenIdApplicationByPostLogoutRedirectUriIndex");
-            SchemaBuilder.DropReduceIndexTable("OpenIdApplicationByRedirectUriIndex");
-            SchemaBuilder.DropReduceIndexTable("OpenIdApplicationByRoleNameIndex");
+            SchemaBuilder.DropReduceIndexTable("OpenIdApplicationByPostLogoutRedirectUriIndex", null);
```

Comment:
@agriffard I would not drop this migration.

You can create a class in the migration with the name, and that should be sufficent 
```
internal class OpenIdApplicationByPostLogoutRedirectUriIndex {}
```

Reply:
ok